app/services/TransactionalService.scala

```diff
@@ -62,6 +62,18 @@ trait TransactionalService {
     }
   }
 
+  def fetchSicCodes(id: String, usePublicAPI: Boolean)(implicit hc: HeaderCarrier): Future[Option[JsValue]] = {
```

Comment:
no incorrect json test, what happens if we get dud sic codes (dud json). we dont want to return a 500 to vat and also we should log it?